### PR TITLE
swupd-client: define OstroXT's default URLs

### PR DESCRIPTION
**Pull request by rojkov**
Configure SWUPD client to use Ostro XT's update repositories by default.

This PR depends on the commit
http://git.yoctoproject.org/cgit/cgit.cgi/meta-swupd/commit/?id=4a970264a0cc6d284f510a0958cf577721e2d082
required to be merged to the ostro-os combo-layer.

### CODE REVIEW

File: meta-ostro-xt/recipes-swupd/swupd-client/swupd-client_%.bbappend [hunk 1]
```diff
@@ -3,6 +3,8 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI_append = " \
     file://ostroprojectorg.key \
 "
+SWUPD_VERSION_URL ?= "https://download.ostroproject.org/updates/ostro-os-xt/milestone/intel-corei7-64/ostro-xt-image-swupd"
```
Comment (mythi) [inline meta-ostro-xt/recipes-swupd/swupd-client/swupd-client_%.bbappend:6]: @rojkov We don't plan to update ostro-os so can you please add the changes in http://git.yoctoproject.org/cgit/cgit.cgi/meta-swupd/commit/?h=pre-sstate&id=606ce9e8e266465619e552489425c7e1a15bec39 here too?

**Comment by mythi** [comment]
@avalluri @igor-stoppa while @rojkov is out, can you make the necessary changes and re-submit?

**Comment by avalluri** [comment]
@mythi  I resubmitted with needed chagnes here: https://github.com/ostroproject/ostro-os-xt/pull/22

**Comment by mythi** [comment]
No longer needed, using #22

**Pull request closed without merging**